cmd/edenConfig.go

```diff
@@ -10,7 +11,11 @@ import (
 )
 
 func newConfigCmd(configName, verbosity *string) *cobra.Command {
```

Comment:
will all these functions be removed anyway once NeoEden tests are done?

Reply:
Not so sure about it,
talking about it on high-level: openevec is your Code As Infrastructure (think terraform) which allows you to describe your environment to deploy EVE, then you have your evetestkit which allows you to access certain states of EVE, Adam and then you have a test, which utilizes openevec and evetestkit to setup certain environment, run certain commands via controller and check EVE state. Openevec can also be used to setup a certain environment via CLI. 
So broadly speaking, I can see scenarios, where I'd like to keep configurations persistent and therefore want to keep it on disk. Talking about it practically, we can't remove it, because changers are dependent on default folder structure and files, in theory we could remove it, propogating configuration there, but that'd be another big refactoring patch, as of now I'd pursue this

Reply:
I mean in more broader terms: can we remove the `eden` commands?

Reply:
but your overview is very nice, thank you :heart:

Reply:
hmmm, that's an interesting take... Do we want Eden commands at all? I'd say yes. Imagine this: you run tests and then it gets stuck or fails, then in order to investigate you just set it up to a certain condition and then you can use eden commands to play around, ssh into thing, change certificates, etc. etc. so you don't waste time writing one openevec command and then  rerunning the whole thing, rather do it interactively